src/detail/Application.hxx

```diff
@@ -138,7 +138,7 @@ Application::gather_stats(opmonlib::InfoCollector& ci, int level)
   ai.state = get_state();
   ai.busy = m_busy.load();
   ai.error = m_error.load();
-  ai.host = boost::asio::ip::host_name();
+  ai.host = getenv("HOSTNAME");
```

Comment:
Probably not needed for HOSTNAME, but it may be a good idea to do a not-NULL check and put it into the std::string constructor
```C++
auto host = getenv("HOSTNAME");
std::string hoststr = "Unknown";
if(host) { hoststr = std::string(host); }
ai.host = hoststr;
```